openmc/examples.py

```diff
@@ -0,0 +1,642 @@
+from numbers import Integral
+
+import numpy as np
+
+import openmc
+import openmc.model
+
+
+def pwr_pin_cell():
+    """Create a PWR pin-cell model.
+
+    This model is a single fuel pin with 2.4 w/o enriched UO2 corresponding to a
+    beginning-of-cycle condition and borated water. The specifications are from
+    the `BEAVRS <http://crpg.mit.edu/research/beavrs>`_ benchmark. Note that the
+    number of particles/batches is initially set very low for testing purposes.
+
+    Returns
+    -------
+    model : openmc.model.Model
+        A PWR pin-cell model
+
+    """
+    model = openmc.model.Model()
+
+    # Define materials.
+    fuel = openmc.Material(name='UO2 (2.4%)')
+    fuel.set_density('g/cm3', 10.29769)
```

Comment:
The quote style used in all the `fuel.set_something(..)` lines are inconsistent. Please use either single quote or double quote consistently at least within related code.